smithy-model/src/main/java/software/amazon/smithy/model/shapes/CollectionShape.java

```diff
@@ -51,18 +59,13 @@ public final MemberShape getMember() {
     }
 
     @Override
-    public final Optional<MemberShape> getMember(String name) {
```

Comment:
Both `members()` and `getMember(name)` are in the current public interface.

Reply:
These are both on and implemented in Shape now